index.html

```diff
@@ -215,6 +215,42 @@ <h5 class="text-white">Skills</h5>
               </ul>
             </div>
         </section>
+      <form action="https://formspree.io/f/xlezlkyw" method="post">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <input name="full-name" type="text" placeholder="Full name" maxlength="30" required>
+        <input name="email" type="email" placeholder="Email address" required>
+        <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] Please add the underline below the name and email input fields and the background color for the text area to make your design match the given design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/186275089-edb3f853-8d03-47b3-93bb-ae0366cea6fc.png)  | ![image](https://user-images.githubusercontent.com/68796232/186275189-070bcb84-7c55-4c72-b57e-c4fbed718fab.png) |